src/cms/src/api/park-photo/content-types/park-photo/lifecycles.js

```diff
@@ -96,28 +96,16 @@ module.exports = {
         )
       }
     }
-    // If parkPhoto.site is selected, get that site.orcsSiteNumber in the parkPhoto.orcsSiteNumber
```

Comment:
I think you meant to delete the two `else if` conditions with `=== null`

Reply:
Oops 😅 I didn't even notice because I was only testing protectedArea ... Thanks!